app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def orders_actors_by_age
+    actors.order(:age)
```

Comment:
Nice encapsulation of this business logic into model methods. I think both of these methods living in the movie model makes total sense